docs/index.md

```diff
@@ -1,91 +1,77 @@
 # Getting Started with Mirascope
 
-Welcome to Mirascope! This guide will help you dive into building powerful LLM-powered applications with ease. We'll start with an interactive tutorial and then guide you through next steps to deepen your knowledge and understanding of Mirascope and all of its features.
+Mirascope: Build powerful LLM applications with ease.
 
-## Interactive Web Search Agent Tutorial
+## Quick Start
 
-Jump right in with our hands-on tutorial where you'll interact with a web search agent built using Mirascope. This will give you a practical taste of Mirascope's key features.
+Install Mirascope and set up your API key:
 
-### Replit Environment
+```bash
+pip install "mirascope[openai]"  # For OpenAI support
+pip install "mirascope[anthropic]"  # For Anthropic support
+# For other providers, use: pip install "mirascope[provider_name]"
+```
 
-We've set up a Replit environment for you to start immediately without any local setup:
+```python
+import os
+os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"
+# For Anthropic: os.environ["ANTHROPIC_API_KEY"] = "YOUR_API_KEY"
+```
+Mirascope supports various LLM providers, including [OpenAI](https://openai.com/), [Anthropic](https://www.anthropic.com/), [Mistral](https://mistral.ai/), [Gemini](https://gemini.google.com), [Groq](https://groq.com/), [Cohere](https://cohere.com/), [LiteLLM](https://www.litellm.ai/), [Azure AI](https://azure.microsoft.com/en-us/solutions/ai), and [Vertex AI](https://cloud.google.com/vertex-ai).
 
-[Mirascope Getting Started Tutorial](https://replit.com/@willbakst/Mirascope-Getting-Started-Tutorial?v=1)
 
-### How to Use the Replit
+Create your first LLM-powered function:
 
-1. Open the Replit link above.
-2. Click the "Fork" button to create your own copy of the Replit.
-3. In the Secrets tab (lock icon on the left sidebar), add your OpenAI API key:
-   - Key: `OPENAI_API_KEY`
-   - Value: Your actual OpenAI API key
-4. Click the "Run" button at the top of the Replit interface.
-5. Interact with the agent in the console on the right side of the screen.
+```python
+from mirascope.core import openai, prompt_template
 
-### Understanding the Code
+@openai.call("gpt-4o-mini")
+@prompt_template("What is the capital of {country}?")
+def get_capital(country: str): ...
 
-The Replit contains a Python script that runs a Web Search Agent implemented using Mirascope, DuckDuckGo Search, BeautifulSoup, and Requests. Here's a breakdown of the key components:
+response = get_capital("Japan")
+print(response.content)
+# Output: The capital of Japan is Tokyo.
+```
 
-- `constants.py`: Constant values used across the various other files.
-- `tools.py`: The `web_search` and `parse_content` tools. These enable the agent to retrieve search results and parse them.
-- `agent.py`: The main `WebSearchAgent` implementation of a streaming agent with access to the web.
-- `main.py`: A simple script that runs the agent.
+Example of streaming responses:
```

Comment:
i think instead of streaming we should instead show structured extraction, saying something like "Easily generate a structured output instead" with the following example:

```python
from mirascope.core import openai, prompt_template
from pydantic import BaseModel

class Book(BaseModel):
    title: str
    author: str

@openai.call("gpt-4o-mini", response_model=Book)
@prompt_template("Extract {book}")
def extract_book(book: str): ...

book = extract_book("The Name of the Wind by Patrick Rothfuss")
assert isinstance(book, Book)
print(book)
# > title='The Name of the Wind' author='Patrick Rothfuss'
```